icpc-wf/ansible/roles/scoreboard/templates/scoreboard.conf.j2

```diff
@@ -0,0 +1,25 @@
+# nginx configuration for the static scoreboard
+server {
+	listen 80;
+	listen [::]:80;
+
+	server_name {{STATIC_SCOREBOARD_HOSTNAME}};
+
+	return 301 https://$host$request_uri;
+}
+
+server {
```

Comment:
Should there be an option where you run the static on only port 80?

Reply:
If we ever need it, sure. We have never done that before so I'd say only add these kinds of things if actually needed?